PancakeSwap-Bots/Set-Masterchef-Settings/src/network.ts

```diff
@@ -0,0 +1,12 @@
+type AgentConfig = Record<number, NetworkData>;
+
+const CONFIG: AgentConfig = {
+  56: {
+    masterChef: "0x73feaa1eE314F8c655E354234017bE2193C9E24E",
+  },
+};
+
```

Comment:
```suggestion
import { Network } from "forta-agent";
type AgentConfig = Record<number, NetworkData>;

const CONFIG: AgentConfig = {
  [Network.BSC]: {
    masterChef: "0x73feaa1eE314F8c655E354234017bE2193C9E24E",
  },
};

```
it's better to import `Network` here and referring to chainId like [Network.BSC], instead of 56

Reply:
Hey, I took this suggestion and changed `56` to `[Network.BSC]` accordingly, but left the testnet (97) as `97` because there is no enum for BSC testnet in `Network`

Reply:
that's the correct way to do it so no problem👍